app/controllers/volunteers_controller.rb

```diff
@@ -1,9 +1,11 @@
 class VolunteersController < ApplicationController
   include NestedAttributes
+  include VolunteerAttributes
   before_action :set_volunteer, only: [:show, :edit, :update, :destroy]
 
   def index
-    @volunteers = Volunteer.all
+    @q = Volunteer.ransack(params[:q] || { state_cont: 'registered' })
```

Comment:
the requirement is that the default filter is like this. And it is still possible to see all. If I do what you suggest, the acceptance criteria is not going to be fullfilled.